src/mds/MDSDaemon.cc

```diff
@@ -1245,6 +1245,7 @@ bool MDSDaemon::handle_core_message(Message *m)
     if (mds_rank) {
       mds_rank->handle_osd_map();
     }
+    m->put();
```

Comment:
@gregsfortytwo Take this one for example. In all the above cases, we are going to return true to indicate we have successfully handled these messages  and therefore are responsible for drop reference of these messages(to get them correctly released). If you take a look at  https://github.com/ceph/ceph/pull/8408/commits/358c81d67786bae27ab45d5f3507d0efb8238026#diff-89e22a97695f436e6645ce25f7bfb985R1223, I think the one is exactly the same.